### PR TITLE
Added AirSense CO2

### PR DESCRIPTION
**Pull request by veneris1991**
Added AirSense CO2 
https://modkam.ru/?p=1715

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -4501,6 +4501,57 @@ const devices = [
         },
         exposes: [e.temperature(), e.humidity(), e.illuminance(), e.illuminance_lux(), e.soil_moisture(), e.pressure(), e.battery()],
     },
+    {
+        zigbeeModel: ['DIYRuZ_AirSense'],
+	    model: 'DIYRuZ_AirSense',
+        vendor: 'DIYRuZ',
+        description: '[Air quality sensor](https://modkam.ru/?p=1715)',
```
Comment (Koenkk) [inline devices.js:4508]: Please add `exposes` here (look at other devices for examples) and fix the lint errors (`npm run lint`).

Comment (veneris1991) [reply devices.js:4517]: Done!

**Comment by veneris1991** [comment]
added exposes and fixed the lint errors

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**